catalyst_voices_packages/catalyst_cardano_serialization/lib/src/exceptions.dart

```diff
@@ -116,14 +116,17 @@ final class TxValueSizeExceededException extends Equatable
 
 /// Exception thrown when parsing a hash that has incorrect length.
 final class HashFormatException extends Equatable implements Exception {
```

Comment:
idea for future. It would be nice to have `CatalystCardanoSerializationException` which implements `Exception`. Its always nice to have one 'catch' no this sort of stuff.

Not for this PR but we can have issue for it

Reply:
Most of these exceptions aren't something you should try to catch, most of them happen when you do a programming mistake or don't validate the data you get from somewhere else. But yeah, there are bunch of others where this applies